internal/pkg/describe/uri.go

```diff
@@ -76,7 +76,7 @@ func (d *LBWebServiceDescriber) URI(envName string) (URI, error) {
 		return URI{}, fmt.Errorf("get stack resources for service %s: %w", d.svc, err)
 	}
 	for _, resource := range resources {
-		if resource.LogicalID == svcStackResourceALBTargetGroupLogicalID {
+		if strings.HasPrefix(resource.LogicalID, svcStackResourceALBTargetGroupLogicalID) {
```

Comment:
Just a reminder that we might need to modify this file for multiple ports  to show multiple urls.

Reply:
So I had this conversation with Efe and Wanxian in the past that if we should be showing all the urls here or just the main url and all three of us had the same choice of showing only main url reason being it looks cluttered if we show all the urls.